assessment/services/assessment.py

```diff
@@ -15,7 +17,7 @@ def get_assessor_or_raise_exception(user: User):
 
 
 def validate_assessment_tool(request_data):
-    if not request_data.get('name'):
+    if not request_data.get('assignment_name'):
```

Comment:
Hello, Indi. I suppose that your commit was overridden by the merging process. I would resolve this after completing the test flow creation.

Reply:
thanks jo, i'll resolve it right now